sirepo/job_supervisor.py

```diff
@@ -312,7 +322,10 @@ def _purge_sim(db_file):
         except Exception as e:
             pkdlog('u={} f={} error={} stack={}', u, f, e, pkdexc())
         finally:
-            cls._purge_free_simulations_set(s, init)
+            tornado.ioloop.IOLoop.current().call_later(
+                _next_call_secs(),
```

Comment:
simplified to `cfg.purge_non_premium_task_period`

Reply:
https://github.com/radiasoft/sirepo/pull/2485/commits/8cb87e0aaa7ead347b40f12da4f6895376ca1e92